src/game.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+
+function Game() {
+  this.rollsTracker = [];
+  this.frameNumber = 1;
+  frameOver = false;
+  left = 10;
```

Comment:
I'm guessing you was going to create a method to count down the pins left that was knocked, I like the thought of that as a way to validate the score.